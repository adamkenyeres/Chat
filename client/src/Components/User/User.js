import React, { Component } from 'react';
import './User.css'
class User extends Component {
    render() {
        const { user } = this.props;
        return (
            <h3 id="user">Your username is: {user} </h3>
        );
    }
}

export default User;
