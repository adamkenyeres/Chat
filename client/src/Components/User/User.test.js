import React from 'react';
import { shallow } from 'enzyme';
import User from './User';


describe('User name', () => {

    it('should render correctly', () => {
        const user = 'Adam';
        const component = shallow(<User user={user} />);

        expect(component).toMatchSnapshot();
    });
});