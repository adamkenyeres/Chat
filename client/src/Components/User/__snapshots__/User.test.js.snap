// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User name should render correctly 1`] = `
<h3
  id="user"
>
  Your username is: 
  Adam
   
</h3>
`;
